diff --git a/apexer/apexer.py b/apexer/apexer.py
--- a/apexer/apexer.py
+++ b/apexer/apexer.py
@@ -675,7 +675,7 @@
   size_in_mb += 100
 
   # Create an empty image
-  cmd = ['/usr/bin/fallocate']
+  cmd = ['fallocate']
   cmd.extend(['-l', str(size_in_mb) + 'M'])
   cmd.append(img_file)
   RunCommand(cmd, args.verbose)
@@ -716,7 +716,7 @@
 
   tmp_input_dir = os.path.join(work_dir, 'tmp_input_dir')
   os.mkdir(tmp_input_dir)
-  cmd = ['/bin/cp', '-ra']
+  cmd = ['cp', '-ra']
   cmd.extend(glob.glob(manifests_dir + '/*'))
   cmd.extend(glob.glob(args.input_dir + '/*'))
   cmd.append(tmp_input_dir)
@@ -749,11 +749,11 @@
 
   # The minimum image size of erofs is 4k, which will cause an error
   # when execute generate_hash_tree in avbtool
-  cmd = ['/bin/ls', '-lgG', img_file]
+  cmd = ['ls', '-lgG', img_file]
   output, _ = RunCommand(cmd, verbose=False)
   image_size = int(output.split()[2])
   if image_size == 4096:
-    cmd = ['/usr/bin/fallocate', '-l', '8k', img_file]
+    cmd = ['fallocate', '-l', '8k', img_file]
     RunCommand(cmd, verbose=False)
 
 
